modules/network-observability-operator-uninstall.adoc

```diff
@@ -9,9 +9,17 @@ You can uninstall the Network Observability Operator using the {product-title} w
 
 .Procedure
 
-. Uninstall the `FlowCollector` custom resource.
+. Remove the `FlowCollector` custom resource.
 .. Click *Flow Collector*, which is next to the *Network Observability Operator* in the *Provided APIs* column. 
 .. Click the options menu {kebab} for the *cluster* and select *Delete FlowCollector*.
 . Uninstall the Network Observability Operator.
 .. Navigate back to the *Operators* -> *Installed Operators* area. 
-.. Click the options menu {kebab} next to the  *Network Observability Operator* and select *Uninstall Operator*.
\ No newline at end of file
+.. Click the options menu {kebab} next to the  *Network Observability Operator* and select *Uninstall Operator*.
+. Remove the `FlowCollector` custom resource definition (CRD).
+.. Navigate to *Administration* -> *CustomResourceDefinitions*. 
+.. Look for *FlowCollector* and click the options menu {kebab}. 
+.. Select *Delete CustomResourceDefinition*.
+[IMPORTANT]
```

Comment:
It doesn't look like the IMPORTANT section is rendering correctly. I think you need a space between line 21 and 22.